package.json

```diff
@@ -50,7 +50,7 @@
     "watchify": "^4.0.0",
     "webpack": "^5.76.0",
     "webpack-cli": "^4.9.1",
-    "wp-rocket-scripts": "^1.0.4",
+    "wp-rocket-scripts": "github:wp-media/rocket-scripts#enhancement/depend-on-php",
```

Comment:
This should be updated with a final version before the release

Reply:
Done